docs/dev_guide/contents/map_rotate_custom.rst

```diff
@@ -0,0 +1,34 @@
+.. _map_rotate_custom:
+
+****************************
+Adding a new rotation method
+****************************
+
+It is possible to select from a number of rotation methods when using :func:`sunpy.image.transform.affine_transform` and :meth:sunpy.map.GenericMap.rotate`.
```

Comment:
```suggestion
It is possible to select from a number of rotation methods when using :func:`sunpy.image.transform.affine_transform` and :meth:`sunpy.map.GenericMap.rotate`.
```

Reply:
I wonder why that didnt raise a warning I could find in the doc build.

Reply:
Heh, I wondered the same thing.  I guess it was so confused by it that it simply interpreted it as straight text.